frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx

```diff
@@ -26,7 +26,7 @@ import {
   WatchK8sResource,
 } from '@console/internal/components/utils/k8s-watch-hook';
 import { useFlag } from '@console/shared/src/hooks/flag';
-import { ProvisionerProps } from '@console/plugin-sdk';
+import { ProvisionerProps } from '@console/dynamic-plugin-sdk/src/extensions/storage-class-provisioner';
```

Comment:
This seems wrong when referencing API from the SDK. It should be exposed at the root.